### ISSUE TITLE
Crash on updating world to 1.9.4

### ISSUE THREAD

**Issue by MentalMouse** [WildBamaBoy/minecraft-comes-alive#731]
- [X] I am running the latest mod versions of MCA and RadixCore.
- [] I can reproduce this issue with just MCA and RadixCore installed.
- [X] I can provide my Minecraft version and MCA version.
- [X] I have searched for the issue previously and it was either not previously reported, or previously fixed and I'm having the same problem.
- [X] I am crashing and can provide my crash report.

## Versions
Forge 1976, MCA-1.9.4-5.2.0-FINAL-universal.jar, mdecore-1.9.4-1.0.jar

Other mod versions:
1.9.4-chunkedgeindicator-1.2.jar
AbyssalCraft-1.9.4-1.9.2.1.jar
Bagginses-1.9.4-2.5.1a.jar
Baubles-1.9.4-1.2.1.0.jar
Chameleon-1.9.4-2.1.5.jar
CyanosLootableBodies_1.9.4-2.1.0.jar
CyclopsCore-1.9.4-0.7.1.jar
EvilCraft-1.9.4-0.10.4.jar
GiacomosFoundry1.9.4-1.3.3.jar
hopperducts-mc1.9.4-1.4.7.jar
jei_1.9.4-3.4.0.204.jar
journeymap-1.9.4-5.2.0-unlimited.jar
LemonsSimpleTeleporters-1.1.2-MC1.9.4.jar
NoMoreRecipeConflict-0.4(1.9).jar
PassthroughSigns-1.9.X-1.0.4.B4-universal.jar
RadixCore-1.9.4-2.1.2-universal.jar
recycler-1.3.0-1.9.4.jar
RoguelikeDungeons-1.9.4-1.5.2.jar
StorageDrawers-1.9.4-3.1.11.jar
telepacks-1.9.4-1.0.jar
WIT-1.9.4-1.0.16-33.jar

ETA Note:  This world was a fresh copy from a working 1.9 world, copied to a new profile with 1.9.4 versions of all mods.

Game crashed during game morning -- I killed a skelly, switched to Vengance Pickaxe, and promptly crashed to desktop.  Since then, the world load consistently crashes, sometimes to desktop, sometimes hanging at "Downloading Terrain".  I originally posted a crashlog on the Evilcraft forum thread (because of the item involvement) here:  http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/1445975-evilcraft-open-source-v0-10-3?comment=607

kroeser responded with:  "I don't see anything in the crashlog indicating EvilCraft's involvement.  The problems seems to be with 'mca.ai.AIWorkday.writeToNBT', so Minecraft Comes Alive is crashing your game."

## Reproduce Steps
The above messages' crashlog is from attempting to reload the world.  Note that this world has a large village which is probably just at the edge of load range from the crash point.

## Additional Information
Here's another crashlog from trying to load the world.

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  MDECore-Core (mdecore-1.9.4-1.0.jar)
Contact their authors BEFORE contacting forge

// Daisy, daisy...

Time: 7/6/16 7:18 PM
Description: Ticking entity

java.lang.NullPointerException: Ticking entity
	at mca.ai.AIWorkday.onUpdateServer(AIWorkday.java:62)
	at mca.ai.AIManager.onUpdate(AIManager.java:45)
	at mca.entity.EntityHuman.func_70071_h_(EntityHuman.java:375)
	at net.minecraft.world.World.func_72866_a(World.java:1951)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:837)
	at net.minecraft.world.World.func_72870_g(World.java:1920)
	at net.minecraft.world.World.func_72939_s(World.java:1743)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:618)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:707)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:611)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:148)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:469)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at mca.ai.AIWorkday.onUpdateServer(AIWorkday.java:62)
	at mca.ai.AIManager.onUpdate(AIManager.java:45)
	at mca.entity.EntityHuman.func_70071_h_(EntityHuman.java:375)
	at net.minecraft.world.World.func_72866_a(World.java:1951)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:837)
	at net.minecraft.world.World.func_72870_g(World.java:1920)

-- Entity being ticked --
Details:
	Entity Type: MCA.EntityHuman (mca.entity.EntityHuman)
	Entity ID: 255
	Entity Name: Nik
	Entity's Exact location: -407.50, 66.00, -106.50
	Entity's Block location: World: (-408,66,-107), Chunk: (at 8,4,5 in -26,-7; contains blocks -416,0,-112 to -401,255,-97), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Entity's Momentum: 0.00, -0.08, 0.00
	Entity's Passengers: []
	Entity's Vehicle: ~~ERROR~~ NullPointerException: null
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1743)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:618)

-- Affected level --
Details:
	Level name: Blood Against the Abyss
	All players: 1 total; [EntityPlayerMP['MentalMouse42'/184, l='Blood Against the Abyss', x=-352.83, y=63.00, z=62.50]]
	Chunk stats: ServerChunkCache: 940 Drop: 0
	Level seed: -8238746004627144004
	Level generator: ID 00 - default, ver 1. Features enabled: true
	Level generator options: 
	Level spawn location: World: (-220,64,228), Chunk: (at 4,4,4 in -14,14; contains blocks -224,0,224 to -209,255,239), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 7353906 game time, 7850968 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 96503 (now: false), thunder time: 7837 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: true
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:707)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:611)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:148)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:469)
	at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
	Minecraft Version: 1.9.4
	Operating System: Linux (amd64) version 4.4.0-28-generic
	Java Version: 1.8.0_91, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 378340640 bytes (360 MB) / 874901504 bytes (834 MB) up to 1559494656 bytes (1487 MB)
	JVM Flags: 5 total; -Xmx1500M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP 9.28 Powered by Forge 12.17.0.1976 27 mods loaded, 26 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.9.4-12.17.0.1976.jar) 
	UCHIJAAAA	Forge{12.17.0.1976} [Minecraft Forge] (forge-1.9.4-12.17.0.1976.jar) 
	UCHIJAAAA	cei{1.2} [Chunk Edge Indicator] (1.9.4-chunkedgeindicator-1.2.jar) 
	UCHIJAAAA	JEI{3.4.0.204} [Just Enough Items] (jei_1.9.4-3.4.0.204.jar) 
	UCHIJAAAA	abyssalcraft{1.9.2.1} [AbyssalCraft] (AbyssalCraft-1.9.4-1.9.2.1.jar) 
	UCHIJAAAA	bagginses{2.4.2d} [Bagginses] (Bagginses-1.9.4-2.5.1a.jar) 
	UCHIJAAAA	Baubles{1.2.1.0} [Baubles] (Baubles-1.9.4-1.2.1.0.jar) 
	UCHIJAAAA	Chameleon{1.9.4-2.1.5} [Chameleon] (Chameleon-1.9.4-2.1.5.jar) 
	UCHIJAAAA	lootablebodies{2.1.0} [DrCyano's Lootable Bodies] (CyanosLootableBodies_1.9.4-2.1.0.jar) 
	UCHIJAAAA	cyclopscore{0.7.1} [Cyclops Core] (CyclopsCore-1.9.4-0.7.1.jar) 
	UCHIJAAAA	evilcraft{0.10.4} [EvilCraft] (EvilCraft-1.9.4-0.10.4.jar) 
	UCHIJAAAA	foundry{1.3.3} [Giacomo's Foundry] (GiacomosFoundry1.9.4-1.3.3.jar) 
	UCHIJAAAA	hopperducts{1.4.7} [Hopper Ducts] (hopperducts-mc1.9.4-1.4.7.jar) 
	UCHIJAAAA	journeymap{1.9.4-5.2.0} [JourneyMap] (journeymap-1.9.4-5.2.0-unlimited.jar) 
	UCHIJAAAA	lteleporters{1.1.2} [Lemon's Simple Teleporters] (LemonsSimpleTeleporters-1.1.2-MC1.9.4.jar) 
	UCHIJAAAA	RadixCore{1.9.4-2.1.2} [RadixCore] (RadixCore-1.9.4-2.1.2-universal.jar) 
	UCHIJAAAA	MCA{1.9.4-5.2.0} [Minecraft Comes Alive] (MCA-1.9.4-5.2.0-FINAL-universal.jar) 
	UCHIJAAAA	mdecore{1.9.4-1.0} [MattDahEpic Core] (mdecore-1.9.4-1.0.jar) 
	UCHIJAAAA	recipehandler{0.4} [NoMoreRecipeConflict] (NoMoreRecipeConflict-0.4(1.9).jar) 
	UCHIJAAAA	passthroughsigns{1.0.4} [Passthrough Signs] (PassthroughSigns-1.9.X-1.0.4.B4-universal.jar) 
	UCHIJAAAA	recycler{1.3} [Corail Recycleur] (recycler-1.3.0-1.9.4.jar) 
	UCHIJAAAA	Roguelike{1.5.2} [Roguelike Dungeons] (RoguelikeDungeons-1.9.4-1.5.2.jar) 
	UCHIJAAAA	StorageDrawers{1.9.4-3.1.11} [Storage Drawers] (StorageDrawers-1.9.4-3.1.11.jar) 
	UCHIJAAAA	telepacks{1.9.4-1.0} [TelePacks] (telepacks-1.9.4-1.0.jar) 
	UCHIJAAAA	WIT{1.0.16} [WIT] (WIT-1.9.4-1.0.16-33.jar) 
	UD	mdecore-core{1.0} [MDE Core - ASM Tweaks] (minecraft.jar) 
	Loaded coremods (and transformers): 
MDECore-Core (mdecore-1.9.4-1.0.jar)
  com.mattdahepic.mdecore.asm.TickrateTransformer
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Player Count: 1 / 8; [EntityPlayerMP['MentalMouse42'/184, l='Blood Against the Abyss', x=-352.83, y=63.00, z=62.50]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'
```

**Comment by MentalMouse** [comment]
Reducing the global chunk-loading distance to 8 chunks (which should certainly cut out the village) doesn't help -- still crashing, snippet below.

`[06:45:20] [Server thread/ERROR]: Failed to save chunk
net.minecraft.util.ReportedException: Saving entity NBT
	at net.minecraft.entity.Entity.func_189511_e(Entity.java:1582) ~[rr.class:?]
	at net.minecraft.entity.Entity.func_70039_c(Entity.java:1491) ~[rr.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75820_a(AnvilChunkLoader.java:322) ~[ast.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75816_a(AnvilChunkLoader.java:173) [ast.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_73242_b(ChunkProviderServer.java:202) [lp.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_186027_a(ChunkProviderServer.java:230) [lp.class:?]
	at net.minecraft.world.WorldServer.func_73044_a(WorldServer.java:1011) [lq.class:?]
	at net.minecraft.server.MinecraftServer.func_71267_a(MinecraftServer.java:354) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71267_a(IntegratedServer.java:215) [byl.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:133) [byl.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:469) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
Caused by: java.lang.NullPointerException
	at mca.ai.AIWorkday.writeToNBT(AIWorkday.java:95) ~[AIWorkday.class:?]
	at mca.ai.AIManager.writeToNBT(AIManager.java:55) ~[AIManager.class:?]
	at mca.entity.EntityHuman.func_70014_b(EntityHuman.java:505) ~[EntityHuman.class:?]
	at net.minecraft.entity.Entity.func_189511_e(Entity.java:1553) ~[rr.class:?]`

**Comment by WildBamaBoy** [comment]
Does this happen on a new world?

**Comment by MentalMouse** [comment]
Not so far -- I'm hoping to test various conditions, but so far I've got a fresh world, spawned a village, and killed a pig.  Being in creative mode, I was able to see it fine and didn't crash.

Then I went exploring, and crashed a new way, when swooping over the graveyards -- it might well have been when I contacted a gravestone.  The world crashed similarly the next time I tried to load it (after the first view of the world), but the third try did load successfully.  I flew out of the graveyard, then moved back in, and crashed again when I got within a block or two of a gravestone.  I haven't been able to load that world in several tries, and I'm about to wipe it and start over.

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  MDECore-Core (mdecore-1.9.4-1.0.jar)
Contact their authors BEFORE contacting forge

// But it works on my machine.

Time: 7/7/16 5:06 PM
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.minecraft.client.renderer.RenderGlobal.func_72731_b(RenderGlobal.java:1938)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1331)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1230)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1039)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1077)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:369)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.minecraft.client.renderer.RenderGlobal.func_72731_b(RenderGlobal.java:1938)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1331)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1230)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['MentalMouse42'/1713, l='MpServer', x=12.48, y=76.65, z=332.73]]
	Chunk stats: MultiplayerChunkCache: 280, 280
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-82,64,306), Chunk: (at 14,4,2 in -6,19; contains blocks -96,0,304 to -81,255,319), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 13128 game time, 13128 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 84 total; [EntitySpider['Spider'/16390, l='MpServer', x=21.50, y=14.00, z=312.50], EntityZombie['Zombie'/16439, l='MpServer', x=31.50, y=18.00, z=266.50], EntityCreeper['Creeper'/16441, l='MpServer', x=41.50, y=27.00, z=399.50], EntityCreeper['Creeper'/16466, l='MpServer', x=36.50, y=33.00, z=289.50], EntityBat['Bat'/16580, l='MpServer', x=-35.32, y=48.26, z=253.32], EntityPig['Pig'/8419, l='MpServer', x=69.37, y=73.00, z=278.26], EntityPlayerSP['MentalMouse42'/1713, l='MpServer', x=12.48, y=76.65, z=332.73], EntitySpider['Spider'/16660, l='MpServer', x=8.50, y=28.00, z=315.50], EntitySkeleton['Skeleton'/16685, l='MpServer', x=-13.50, y=28.00, z=363.50], EntitySkeleton['Skeleton'/16686, l='MpServer', x=-37.50, y=46.00, z=255.50], EntitySkeleton['Skeleton'/16727, l='MpServer', x=68.50, y=21.00, z=305.50], EntityZombie['Zombie'/16745, l='MpServer', x=76.50, y=11.00, z=300.50], EntityCreeper['Creeper'/16748, l='MpServer', x=36.50, y=13.00, z=314.50], EntitySkeleton['Skeleton'/16763, l='MpServer', x=-37.50, y=29.00, z=308.50], EntityBat['Bat'/16797, l='MpServer', x=-15.39, y=31.58, z=372.06], EntityBat['Bat'/16798, l='MpServer', x=-48.00, y=39.67, z=392.69], EntityBat['Bat'/16819, l='MpServer', x=78.34, y=50.20, z=353.72], EntitySkeleton['Skeleton'/16828, l='MpServer', x=-25.50, y=33.00, z=405.50], EntityZombie['Zombie'/16849, l='MpServer', x=25.50, y=21.00, z=263.50], EntityBat['Bat'/16900, l='MpServer', x=21.70, y=37.10, z=364.59], EntityZombie['Zombie'/16926, l='MpServer', x=-7.50, y=14.00, z=391.50], EntitySkeleton['Skeleton'/16948, l='MpServer', x=32.50, y=14.00, z=254.50], EntityPig['Pig'/568, l='MpServer', x=-46.50, y=73.00, z=341.68], EntityPig['Pig'/569, l='MpServer', x=-19.24, y=76.00, z=345.45], EntityPig['Pig'/570, l='MpServer', x=-35.37, y=76.00, z=331.76], EntityPig['Pig'/571, l='MpServer', x=-53.26, y=72.00, z=328.54], EntityPig['Pig'/574, l='MpServer', x=-42.31, y=76.00, z=284.50], EntityPig['Pig'/576, l='MpServer', x=-24.55, y=76.00, z=319.55], EntityPig['Pig'/577, l='MpServer', x=-24.47, y=76.00, z=345.32], EntityChicken['Chicken'/579, l='MpServer', x=-29.50, y=70.00, z=402.50], EntityChicken['Chicken'/580, l='MpServer', x=-30.50, y=73.00, z=405.50], EntityChicken['Chicken'/581, l='MpServer', x=-29.50, y=71.00, z=407.50], EntityPig['Pig'/587, l='MpServer', x=-35.23, y=76.00, z=321.50], EntityCreeper['Creeper'/14925, l='MpServer', x=54.50, y=24.00, z=380.50], EntityEnderman['Enderman'/16988, l='MpServer', x=29.50, y=13.00, z=306.50], EntityCreeper['Creeper'/17002, l='MpServer', x=10.50, y=16.00, z=322.50], EntityPig['Pig'/8832, l='MpServer', x=65.80, y=73.00, z=274.73], EntityPig['Pig'/8833, l='MpServer', x=66.50, y=73.00, z=276.50], EntityPig['Pig'/8834, l='MpServer', x=65.41, y=74.00, z=274.30], EntitySkeleton['Skeleton'/17038, l='MpServer', x=9.50, y=20.00, z=257.50], EntityCreeper['Creeper'/17048, l='MpServer', x=-1.53, y=34.00, z=322.82], EntityZombie['Zombie'/17089, l='MpServer', x=12.50, y=58.00, z=406.50], EntityZombie['Zombie'/17104, l='MpServer', x=69.49, y=31.00, z=382.61], EntityCreeper['Creeper'/17116, l='MpServer', x=47.50, y=60.00, z=382.50], EntityCreeper['Creeper'/17118, l='MpServer', x=39.50, y=43.00, z=369.50], EntitySquid['Squid'/17138, l='MpServer', x=-51.08, y=60.00, z=356.26], EntityZombie['Zombie'/17149, l='MpServer', x=43.50, y=35.00, z=295.50], EntityZombie['Zombie'/17151, l='MpServer', x=-16.50, y=16.00, z=389.50], EntityCreeper['Creeper'/17152, l='MpServer', x=83.50, y=39.00, z=344.50], VengeanceSpirit['Vengeance Spirit'/17178, l='MpServer', x=21.76, y=12.61, z=258.29], EntityBat['Bat'/17180, l='MpServer', x=78.19, y=21.64, z=300.58], EntityBat['Bat'/17190, l='MpServer', x=-53.00, y=45.04, z=268.15], EntityEnderman['Enderman'/11141, l='MpServer', x=87.50, y=20.00, z=349.50], EntityZombie['Zombie'/15308, l='MpServer', x=80.79, y=43.00, z=348.56], EntityPig['Pig'/9241, l='MpServer', x=91.50, y=79.00, z=330.50], EntityZombie['Zombie'/15561, l='MpServer', x=-20.50, y=29.00, z=377.50], EntitySkeleton['Skeleton'/15596, l='MpServer', x=62.50, y=33.00, z=378.50], EntitySkeleton['Skeleton'/15598, l='MpServer', x=8.50, y=63.00, z=313.79], EntityZombie['Zombie'/15688, l='MpServer', x=12.50, y=18.00, z=401.50], EntitySkeleton['Skeleton'/15836, l='MpServer', x=52.50, y=12.00, z=267.50], EntitySkeleton['Skeleton'/15863, l='MpServer', x=50.50, y=27.00, z=372.50], EntityZombie['Zombie'/15884, l='MpServer', x=80.50, y=19.00, z=346.50], EntityHuman['Bong'/9777, l='MpServer', x=17.00, y=77.00, z=299.00], EntityHuman['Toby'/9779, l='MpServer', x=9.00, y=77.00, z=310.00], EntityHuman['Lilly'/9780, l='MpServer', x=17.00, y=77.00, z=320.00], EntityHuman['Liliana'/9782, l='MpServer', x=17.00, y=77.00, z=321.00], EntityHuman['Launa'/9785, l='MpServer', x=17.00, y=77.00, z=321.00], EntityHuman['Elinore'/9786, l='MpServer', x=-9.17, y=76.00, z=300.09], EntityHuman['Sergio'/9787, l='MpServer', x=-6.91, y=76.00, z=314.82], EntityHuman['Maurita'/9788, l='MpServer', x=-12.00, y=76.00, z=329.00], EntityHuman['Carlita'/9789, l='MpServer', x=-6.00, y=76.00, z=329.00], EntityHuman['Willian'/9790, l='MpServer', x=-6.00, y=76.00, z=328.00], EntityHuman['Floy'/9791, l='MpServer', x=-14.75, y=76.00, z=306.43], EntityHuman['Millard'/9792, l='MpServer', x=0.14, y=76.00, z=329.15], EntityBat['Bat'/15965, l='MpServer', x=32.50, y=45.10, z=409.03], EntityWitch['Witch'/13948, l='MpServer', x=7.50, y=29.00, z=324.50], EntityZombie['Zombie'/16096, l='MpServer', x=52.50, y=43.00, z=374.50], EntitySpider['Spider'/16107, l='MpServer', x=-2.99, y=47.00, z=276.43], EntityBat['Bat'/16160, l='MpServer', x=3.25, y=48.10, z=281.74], EntitySpider['Spider'/16190, l='MpServer', x=41.50, y=18.00, z=257.50], EntityCreeper['Creeper'/14300, l='MpServer', x=87.50, y=41.00, z=335.50], EntityBat['Bat'/16358, l='MpServer', x=82.58, y=39.10, z=347.25], EntityHuman['Kallie'/10235, l='MpServer', x=12.30, y=77.00, z=298.30], EntityHuman['Otilia'/10236, l='MpServer', x=-0.49, y=76.00, z=313.38]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:412)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2651)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.9.4
	Operating System: Linux (amd64) version 4.4.0-28-generic
	Java Version: 1.8.0_91, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 218664392 bytes (208 MB) / 1015717888 bytes (968 MB) up to 1559494656 bytes (1487 MB)
	JVM Flags: 5 total; -Xmx1500M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP 9.28 Powered by Forge 12.17.0.1976 27 mods loaded, 26 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAAAAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAAAAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.9.4-12.17.0.1976.jar) 
	UCHIJAAAAAAAAA	Forge{12.17.0.1976} [Minecraft Forge] (forge-1.9.4-12.17.0.1976.jar) 
	UCHIJAAAAAAAAA	cei{1.2} [Chunk Edge Indicator] (1.9.4-chunkedgeindicator-1.2.jar) 
	UCHIJAAAAAAAAA	JEI{3.4.0.204} [Just Enough Items] (jei_1.9.4-3.4.0.204.jar) 
	UCHIJAAAAAAAAA	abyssalcraft{1.9.2.1} [AbyssalCraft] (AbyssalCraft-1.9.4-1.9.2.1.jar) 
	UCHIJAAAAAAAAA	bagginses{2.4.2d} [Bagginses] (Bagginses-1.9.4-2.5.1a.jar) 
	UCHIJAAAAAAAAA	Baubles{1.2.1.0} [Baubles] (Baubles-1.9.4-1.2.1.0.jar) 
	UCHIJAAAAAAAAA	Chameleon{1.9.4-2.1.5} [Chameleon] (Chameleon-1.9.4-2.1.5.jar) 
	UCHIJAAAAAAAAA	lootablebodies{2.1.0} [DrCyano's Lootable Bodies] (CyanosLootableBodies_1.9.4-2.1.0.jar) 
	UCHIJAAAAAAAAA	cyclopscore{0.7.1} [Cyclops Core] (CyclopsCore-1.9.4-0.7.1.jar) 
	UCHIJAAAAAAAAA	evilcraft{0.10.4} [EvilCraft] (EvilCraft-1.9.4-0.10.4.jar) 
	UCHIJAAAAAAAAA	foundry{1.3.3} [Giacomo's Foundry] (GiacomosFoundry1.9.4-1.3.3.jar) 
	UCHIJAAAAAAAAA	hopperducts{1.4.7} [Hopper Ducts] (hopperducts-mc1.9.4-1.4.7.jar) 
	UCHIJAAAAAAAAA	journeymap{1.9.4-5.2.0} [JourneyMap] (journeymap-1.9.4-5.2.0-unlimited.jar) 
	UCHIJAAAAAAAAA	lteleporters{1.1.2} [Lemon's Simple Teleporters] (LemonsSimpleTeleporters-1.1.2-MC1.9.4.jar) 
	UCHIJAAAAAAAAA	RadixCore{1.9.4-2.1.2} [RadixCore] (RadixCore-1.9.4-2.1.2-universal.jar) 
	UCHIJAAAAAAAAA	MCA{1.9.4-5.2.0} [Minecraft Comes Alive] (MCA-1.9.4-5.2.0-FINAL-universal.jar) 
	UCHIJAAAAAAAAA	mdecore{1.9.4-1.0} [MattDahEpic Core] (mdecore-1.9.4-1.0.jar) 
	UCHIJAAAAAAAAA	recipehandler{0.4} [NoMoreRecipeConflict] (NoMoreRecipeConflict-0.4(1.9).jar) 
	UCHIJAAAAAAAAA	passthroughsigns{1.0.4} [Passthrough Signs] (PassthroughSigns-1.9.X-1.0.4.B4-universal.jar) 
	UCHIJAAAAAAAAA	recycler{1.3} [Corail Recycleur] (recycler-1.3.0-1.9.4.jar) 
	UCHIJAAAAAAAAA	Roguelike{1.5.2} [Roguelike Dungeons] (RoguelikeDungeons-1.9.4-1.5.2.jar) 
	UCHIJAAAAAAAAA	StorageDrawers{1.9.4-3.1.11} [Storage Drawers] (StorageDrawers-1.9.4-3.1.11.jar) 
	UCHIJAAAAAAAAA	telepacks{1.9.4-1.0} [TelePacks] (telepacks-1.9.4-1.0.jar) 
	UCHIJAAAAAAAAA	WIT{1.0.16} [WIT] (WIT-1.9.4-1.0.16-33.jar) 
	UD	mdecore-core{1.0} [MDE Core - ASM Tweaks] (minecraft.jar) 
	Loaded coremods (and transformers): 
MDECore-Core (mdecore-1.9.4-1.0.jar)
  com.mattdahepic.mdecore.asm.TickrateTransformer
	GL info: ' Vendor: 'X.Org' Version: '3.0 Mesa 11.2.0' Renderer: 'Gallium 0.4 on AMD ARUBA (DRM 2.43.0, LLVM 3.8.0)'
	Launched Version: 1.9.4-forge1.9.4-12.17.0.1976
	LWJGL: 2.9.4
	OpenGL: Gallium 0.4 on AMD ARUBA (DRM 2.43.0, LLVM 3.8.0) GL version 3.0 Mesa 11.2.0, X.Org
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: net.minecraft.client.Minecraft$15@1268c120

```

**Comment by MentalMouse** [comment]
I managed to verify that in a new world it also didn't crash for naturally-generated villagers.  At that point, I gave up and loaded the world without MCA (deleting all the villagers), spawned in a few new villages, then put MCA back in.  The new villagers do not seem crashy.
